RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py

```diff
@@ -167,6 +168,31 @@ def customizeHLTIter0ToMkFit(process):
 
     return process
 
+def customizeHLTStripHitsFromMkFit(process):
```

Comment:
this eventually needs to become a customization routing in `customizeHLTforCMSSW` once mkFit is included in the regular menu and needs to be activated by default.